### DISCUSSION TITLE
CHIA NODE not synced constantly

### DISCUSSION THREAD

**Discussion by jordanr12345** [Chia-Network/chia-blockchain#4973] [Support]
Chia Full node Status always `**NOT SYNCED.**` What should I do? 
I have tried everything from opening ports too redownloading CHIA block chain.

I have CHIA program open on my desktop for 4 full days now and still nothing.

Please advise

**Comment by forrestcamps** [comment]
NOT SYNCED appears here always, after the 1.1.5 version, I go to sleep and when a woke up is Not Synced : ( Plotting stoped.

**Comment by SackofFluor** [comment]
yeah... since this morning all my nodes disconnected and I cant get any of them to sync anymore
fuck this BS

**Reply by forrestcamps** [reply]
I'm getting insane... A new bug every day. Probably It's me the problem. I'm noob.

**Comment by SackofFluor** [comment]
No idea... So far it's getting worse by the day.... Slowly I have the feeling that there are less then 25% of actually valid, farming plots... The rest are useful idiots like myself...

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
